src/App/Lobby/Room.tsx

```diff
@@ -47,7 +48,7 @@ export class Room extends React.Component<IRoomProps, IRoomState> {
       return <LoadingPage />;
     }
     if (!nickname) {
-      return this._getNamePrompt();
+      return <FreeBoardGameBar>{this._getNamePrompt()}</FreeBoardGameBar>;
```

Comment:
Any reason to move the FBG bar here? Can you move back?

Reply:
I didn't want to include the FBG bar in _getNamePrompt() because I don't always want the fbg bar.

https://github.com/freeboardgame/FreeBoardGame.org/blob/d46a426cf36401f5c640cb93071b0e387cc7d748/src/App/Lobby/Room.tsx#L65

Reply:
If you are reusing everything on _getNamePrompt, why don't you move this to the change nickname component instead?